lib/devise/multi_email/models/authenticatable.rb

```diff
@@ -69,27 +52,18 @@ def find_first_by_auth_conditions(tainted_conditions, opts = {})
 
           if email && email.is_a?(String)
             conditions = filtered_conditions.to_h.merge(opts).
-              reverse_merge(emails: { email: email })
+              reverse_merge(multi_email_association.reflection.table_name => { email: email })
```

Comment:
@allenwq interesting that it worked fine for you. It could maybe be that the association you have is setup so that the properly generated SQL infers the right table names.

However, I've run into issues with custom tables, and this StackOverflow post describes the issue as well: http://stackoverflow.com/questions/36546960/joins-and-where-request-on-association-with-custom-table-name